app/post.go

```diff
@@ -866,12 +866,12 @@ func (a *App) GetSinglePost(postID string, includeDeleted bool) (*model.Post, *m
 		}
 	}
 
-	isInaccessible, appErr := a.isInaccessiblePost(post)
+	isInaccessible, firstInaccessiblePostTime, appErr := a.isInaccessiblePost(post)
 	if appErr != nil {
 		return nil, appErr
 	}
 	if isInaccessible {
-		return nil, model.NewAppError("GetSinglePost", "app.post.cloud.get.app_error", nil, "", http.StatusForbidden)
+		return nil, model.NewAppError("GetSinglePost", "app.post.cloud.get.app_error", map[string]any{"firstInaccessiblePostTime": firstInaccessiblePostTime}, "", http.StatusForbidden)
```

Comment:
You're passing the `map` here but it's not being used/logged anywhere in the `app.post.cloud.get.app_error` message.

Reply:
Good catch! That was from an earlier idea I had that didn't turn out well. Removed.